apps/test/unit/applab/exportTest.js

```diff
@@ -0,0 +1,29 @@
+import {assert} from '../../util/configuredChai';
+import {blocks} from '@cdo/apps/applab/dropletConfig';
+import {getExportedGlobals} from '@cdo/apps/applab/export';
+
+describe("the export module", () => {
+  describe("the getExportedGlobals function", () => {
+    let globals;
+    before(() => {
+      globals = Object.keys(getExportedGlobals());
+    });
+    blocks.forEach(block => {
+      if (block.blockPrefix || block.block) {
+        // we don't care about functions defined on objects other than window
+        // for example: Array.indexOf. We also don't care about this with blocks,
+        // like declareAssign_str_hello_world, which is just:
+        //
+        //   var str = "Hello World";
+        return;
+      }
+      it(`exposes a ${block.func} function`, () => {
+        if (globals.indexOf(block.func) < 0) {
+          if (!eval(`window.${block.func}`)) { //eslint-disable-line no-eval
+            assert(false, `expected exported globals to include ${block.func}`);
+          }
+        }
+      });
```

Comment:
This is pretty sweet - requiring internal consistency between the `dropletConfig` and the exported globals.  But there's something I don't understand about the export process here - do these necessarily need to be global?  In `dropletConfig` most entries have a `parent` object where they'll supposedly find the method they're looking for, without it being on `window` AFAIK.